docs/maintainers/issue-management.md

```diff
@@ -62,15 +62,6 @@ These are for issues that are specific to certain Operating Systems:
 
 ### Process Tags
 
-Issues can be in one of the following states:
-* `0 - Backlog` : We've acknowledged the issue but have no immediate plans to address it.
-  If you're looking for a way to contribute, these issues can be a good place to start.
-* `1 - Planning` : The issue requires some design or discussion before coding can begin.
-* `2 - Ready` : Any design or discussion is essentially done, coding has not yet begun.
-* `3 - Working` : The assignee(s) are actively working on the issue.
-* `4 - In Review` : The issue is being reviewed.
-  The assignee(s) are responsible for signing off before the PR will be merged.
-
 The following labels are used on PRs:
 
 * `Review - Needed` : The PR is being reviewed.  Please see [Pull Request - Code Review](../../.github/CONTRIBUTING.md#pull-request-code-review)
```

Comment:
Sorry, it is not from the PR but here link to `Pull Request - Code Review` is broken. 
Right link is `https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---code-review`